third_party/3/pkg_resources.pyi

```diff
@@ -226,12 +226,12 @@ def get_provider(package_or_requirement: str) -> IResourceProvider: ...
 def get_provider(package_or_requirement: Requirement) -> Distribution: ...
 
 class IMetadataProvider:
```

Comment:
This class is weird. Its source code really doesn't have 'self' parameters. I suppose this is implicitly using zope.Interfaces (but not explicitly -- in fact it seems it's just documentation at this point). I wonder if we should just remove these 'I' classes from this package in typeshed -- if anything explicitly references them, it probably won't work.

Maybe we should ask the pypa folks, and leave this file alone for now.